Altis_Life.Altis/core/items/fn_boltcutter.sqf

```diff
@@ -6,48 +6,68 @@
+private ["_building", "_door", "_doors", "_cpRate", "_title", "_progressBar", "_titleText", "_cp", "_ui"];
+_building = param [0, objNull, [objNull]];
+private _home = false;
+private _ownerName = "";
 
 private _vaultHouse = [[["Altis", "Land_Research_house_V1_F"], ["Tanoa", "Land_Medevac_house_V1_F"]]] call TON_fnc_terrainSort;
-private _altisArray = [16019.5,16952.9,0];
-private _tanoaArray = [11074.2,11501.5,0.00137329];
+private _altisArray = [16019.5, 16952.9, 0];
+private _tanoaArray = [11074.2, 11501.5, 0.00137329];
 private _pos = [[["Altis", _altisArray], ["Tanoa", _tanoaArray]]] call TON_fnc_terrainSort;
 
 if (isNull _building) exitWith {};
 if (!(_building isKindOf "House_F")) exitWith {hint localize "STR_ISTR_Bolt_NotNear";};
-if (((nearestObject [_pos,"Land_Dome_Big_F"]) == _building || (nearestObject [_pos,_vaultHouse]) == _building) && (west countSide playableUnits < (LIFE_SETTINGS(getNumber,"minimum_cops")))) exitWith {
-    hint format [localize "STR_Civ_NotEnoughCops",(LIFE_SETTINGS(getNumber,"minimum_cops"))];
+if (((nearestObject [_pos, "Land_Dome_Big_F"]) == _building || (nearestObject [_pos, _vaultHouse]) == _building) && (west countSide playableUnits < (LIFE_SETTINGS(getNumber, "minimum_cops")))) exitWith {
+    hint format [localize "STR_Civ_NotEnoughCops", (LIFE_SETTINGS(getNumber, "minimum_cops"))];
 };
-if ((typeOf _building) == _vaultHouse && (nearestObject [_pos,"Land_Dome_Big_F"]) getVariable ["locked",true]) exitWith {hint localize "STR_ISTR_Bolt_Exploit"};
+if ((typeOf _building) == _vaultHouse && (nearestObject [_pos, "Land_Dome_Big_F"]) getVariable ["locked", true]) exitWith {hint localize "STR_ISTR_Bolt_Exploit"};
 if (isNil "life_boltcutter_uses") then {life_boltcutter_uses = 0;};
 
-_doors = FETCH_CONFIG2(getNumber,"CfgVehicles",(typeOf _building),"numberOfDoors");
+_doors = FETCH_CONFIG2(getNumber, "CfgVehicles", (typeOf _building), "numberOfDoors");
 _door = 0;
 //Find the nearest door
 for "_i" from 1 to _doors do {
-    _selPos = _building selectionPosition format ["Door_%1_trigger",_i];
+    _selPos = _building selectionPosition format ["Door_%1_trigger", _i];
     _worldSpace = _building modelToWorld _selPos;
         if (player distance _worldSpace < 2) exitWith {_door = _i;};
 };
 if (_door isEqualTo 0) exitWith {hint localize "STR_Cop_NotaDoor"}; //Not near a door to be broken into.
-if ((_building getVariable [format ["bis_disabled_Door_%1",_door],0]) isEqualTo 0) exitWith {hint localize "STR_House_Raid_DoorUnlocked"};
+if ((_building getVariable [format ["bis_disabled_Door_%1", _door], 0]) isEqualTo 0) exitWith {hint localize "STR_House_Raid_DoorUnlocked"};
 
-if ((nearestObject [_pos,"Land_Dome_Big_F"]) == _building || (nearestObject [_pos,_vaultHouse]) == _building) then {
-    [[1,2],"STR_ISTR_Bolt_AlertFed",true,[]] remoteExecCall ["life_fnc_broadcast",RCLIENT];
+if ((nearestObject [_pos, "Land_Dome_Big_F"]) == _building || (nearestObject [_pos, _vaultHouse]) == _building) then {
```

Comment:
See [this comment](https://github.com/AsYetUntitled/Framework/pull/73/files#r97928005).